docker-compose-remote.yml

```diff
@@ -71,5 +71,8 @@ services:
           cpus: '0.10'
           memory: 200M
 
+  pg:
+    restart: always
+
```

Comment:
There should be resource allocation specified here similar to the other containers above.